### PR TITLE
Document requirements

### PR DESCRIPTION
**Pull request by greg0ire**
Refs https://github.com/api-platform/core/pull/1406

### CODE REVIEW

File: core/operations.md [hunk 1]
```diff
@@ -128,8 +128,8 @@ use ApiPlatform\Core\Annotation\ApiResource;
 
 /**
  * @ApiResource(itemOperations={
- *     "get"={"method"="GET", "path"="/grimoire/{id}"},
- *     "put"={"method"="PUT", "path"="/grimoire/{id}/update", "hydra_context"={"foo"="bar"},
```
Comment (greg0ire) [inline core/operations.md]: Is that kind of trailing comma really supported?

Comment (soyuka) [reply core/operations.md]: why not :P

Comment (greg0ire) [reply core/operations.md]: Oh right it's just an array after all...

**Comment by meyerbaptiste** [comment]
Thanks @greg0ire

**Pull request merged**